packages/apps/graph/generated-schema.graphql

```diff
@@ -168,21 +191,50 @@ type PageInfo {
 scalar PositiveFloat
 
 type Query {
-  account(accountName: String!, moduleName: String!): ModuleAccount!
+  """Find an account by its name and its module, such as coin."""
+  account(accountName: String!, moduleName: String!): ModuleAccount
+
+  """Find a block by its hash."""
   block(hash: String!): Block
+
+  """Find all blocks from a given height."""
   blocksFromHeight(chainIds: [String!], startHeight: Int!): [Block!]!
+
+  """
+  Find an account by its name and its module, such as coin, on a specific chain.
+  """
   chainAccount(accountName: String!, chainId: String!, moduleName: String!): ChainModuleAccount
+
+  """Find all completed blocks from a given height."""
   completedBlockHeights(chainIds: [String!], completedHeights: Boolean, heightCount: Int): [Block!]!
+
+  """Estimate the gas limit for a transaction."""
   gasLimitEstimate(transaction: PactTransaction!): Int!
+
+  """Estimate the gas limit for a list of transactions."""
   gasLimitEstimates(transactions: [PactTransaction!]!): [Int!]!
+
+  """Get the configuration of the graph."""
   graphConfiguration: GraphConfiguration!
+
+  """Get the height of the latest block."""
```

Comment:
```suggestion
  """Get the height of the block with the highest height."""
```
It's a subtle difference, but we don't retreive the `last` block height, but the highest block height